src/components/WithdrawPage.tsx

```diff
@@ -95,55 +108,58 @@ const WithdrawPage = (props: Props): ReactElement => {
           >
             <div className="form">
               <h3>{t("withdraw")}</h3>
-              <div className="percentage">
-                <span>{`${t("withdrawPercentage")} (%):`}</span>
-                <input
+              <Box display="flex">
+                <Box>
```

Comment:
does it look bad without this `Box`?

Reply:
Yes, because of the width of `TextField`, the Typography is overflowing if it doesn't be wrapped with `div` or `Box`.

![image](https://user-images.githubusercontent.com/75422800/153918370-5eabbddd-0972-4082-a60e-b450b74e5f2d.png)

Reply:
great thanks for double checking that